Documentation/flex-to-csi-migration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Flex Migration
+weight: 11900
+indent: true
+---
+
+# Flex to CSI Migration
+
+In Rook v1.8, the Flex driver has been deprecated. Before updating to v1.8, any Flex volumes created in previous versions of Rook will need to be converted to Ceph-CSI volumes.
+
+The tool [persistent-volume-migrator](https://github.com/ceph/persistent-volume-migrator) will help automate migration of Flex rbd volumes to Ceph-CSI volumes.
```

Comment:
A note about CephFS not being supported?

Reply:
added